mono/dis/get.c

```diff
@@ -1854,7 +1856,7 @@ get_field (MonoImage *m, guint32 token, MonoGenericContainer *container)
 }
 
 static MonoGenericContainer *
-get_memberref_container (MonoImage *m, guint32 mrp_token, MonoGenericContainer *container)
+get_memberref_container (MonoImage *m, guint32 mrp_token, MonoGenericContainer *container, MonoError *error)
```

Comment:
We're adding MonoError but keeping it ok when nothing is found.
This non-conforming behavior (ok and null return) must be either addressed or documented.